src/static/helpers/IDPLoginHelper.ts

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
I separated this helper out into another file, as this repo has a 500 line limit - adding this to `slasHelper.ts` would violate this rule.

Reply:
I think there's a good argument for removing this 500 line limit considering that this helper makes sense to be within `slasHelper.ts`, alongside the other helper functions that will be implemented. Let's continue the conversation here:

https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/pull/173#discussion_r1771757959

Reply:
Just saw your comment - will revise as discussed!